.gitignore

```
@@ -1,3 +1,4 @@
 .idea*
 *.iml
 /target*
+/file_with_logs*
```

```suggestion
/folder_with_logs*
```
краще змінити назву папки